content/en/blog/2023/sig-release-spotlight/sig-release-spotlight.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: blog
+title: "Spotlight on SIG Release (Release Team Subproject)"
+date: 2023-08-29
+slug: sig-release-spotlight-2023
+author: Nitish Kumar
+---
+
+**Author**: Nitish Kumar
+
+The Release Special Interest Group  (SIG Release), where Kubernetes sharpens its blade with cutting-edge features and bug fixes every 4 months:  have you ever considered how such a big project like Kubernetes manages its timeline so efficiently to release its new version,  or how the internal workings of the Release Team look like? If you're curious about these questions or want to know more and get involved with the work SIG release does, read on!
+
+SIG Release plays a crucial role in the development and evolution of Kubernetes. Its primary responsibility is to manage the release process of new versions of Kubernetes. It operates on a regular release cycle, [typically every three to four months](https://kubernetes.io/releases/release/). During this cycle, the Kubernetes Release Team works closely with other SIGs and contributors to ensure a smooth and well-coordinated release. This includes planning the release schedule, setting deadlines for code freeze and testing phases, as well as creating release artefacts like binaries, documentation, and release notes.
+
+Before you read further, it is important to note that there are two subprojects under SIG Release, _Release Engineering_ and _Release Team_.
+
+In this blog post, [Nitish Kumar](https://twitter.com/Nitishtwt06) interviews the [Chairs and Technical Leads](https://github.com/kubernetes/community/tree/master/sig-release#leadership) of SIG Release with the spotlight on the Release Team subproject, how the release process looks like, and ways to get involved.
```

Comment:
I would name who those interviewees were _at the time of the interview_, as https://github.com/kubernetes/community/tree/master/sig-release#leadership is subject to change.

Reply:
This was the response from Veronica:

_I am the only one who typed the answers, however, my answers are based on the responses we tend to give as a group in other similar situations, such as interviews, blog posts and talks. Most of the team members were on vacation from the beginning of the summer, and others are currently on vacation, so taking everyone's thoughts can be tricky. But my suggestion is to ask one question per available team member and see what they think.
Similar questions and info can be found in these videos:_

Reply:
@nitishfy I would use something like "... interviews Verónica López (PlanetScale), Technical Lead of SIG Release". This is objectively true since being interviewed doesn't mean that the information conveyed wasn't originally part of a group effort (that's almost always true). If it's absolutely needed, you can add "... of SIG Release (with assistance from Foo and Bar, also Leads in SIG Release).